datetime.fifthtry.site/js/functions.js

```diff
@@ -2,29 +2,64 @@ function datetime(diff) {
     const currentDate = new Date();
```

Comment:
Can you use IIFE pattern, and expose a single top level symbol, `window.fastn_datetime`, so all other stuff are visible as `window.fastn_datetime.constructSingleInt` etc.